packages/forms-web-app/src/dynamic-forms/dynamic-components/macros/answer-list.njk

```diff
@@ -15,12 +15,12 @@
             actions: {
                 items: [
                     {
-                    href: item.removeLink,
-                    text: "Remove",
-                    visuallyHiddenText: "remove"
+                        href: item.removeLink,
+                        text: "Remove",
+                        visuallyHiddenText: item.label + ", " + item.answer
```

Comment:
Does the screen reader read "Remove" and then follows it with the visually hidden text?

Reply:
the govuk macro ends up with:
<a class="govuk-link" href="...">
  Remove<span class="govuk-visually-hidden"> {Label}, {answer} </span>
</a>

e.g.:
<a class="govuk-link" href="...">
 Remove<span class="govuk-visually-hidden"> Neighbour 1, Test, TestTown, wn1 1ab </span>
</a>


I can't download NVDA on this device but logically should read out the text in the link